# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/fwbuilder/errno.diff
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN fwbuilder-2.0.3/src/gui/SSHPIX.cpp fwbuilder-2.0.3-new/src/gui/SSHPIX.cpp
--- fwbuilder-2.0.3/src/gui/SSHPIX.cpp	2004-09-29 08:31:52.000000000 +0200
+++ fwbuilder-2.0.3-new/src/gui/SSHPIX.cpp	2004-10-03 22:55:51.801233872 +0200
@@ -40,6 +40,11 @@
 
 #include <iostream>
 
+#include <errno.h>
+#ifndef errno
+extern int errno;
+#endif 
+
 using namespace std;
 
 SSHPIX::SSHPIX(const QString &_h,
